helix-term/src/ui/editor.rs

```diff
@@ -409,29 +414,54 @@ impl EditorView {
         let base_cursor_scope = theme
             .find_scope_index_exact("ui.cursor")
             .unwrap_or(selection_scope);
+        let base_cursor_unfocused_scope = theme
+            .find_scope_index_exact("ui.cursor_unfocused")
```

Comment:
These theme keys with `_unfocused` should be `*.unfocused` instead. `ui.cursor.primary_unfocused.{insert,select,normal}` should be `ui.cursor.primary.{insert,select,normal}.unfocused`.

The new theme keys should be added to the book in `book/src/themes.md`

Reply:
Changed the names and added the keys into the book. I also removed non primary keys as I only update the primary cursor.